Source/MSBuild.Sdk.Extras/Sdk/Sdk.props

```diff
@@ -22,7 +22,7 @@
   
   <Import Condition=" '$(_ExtrasHasDesktopSdk)' != 'true' " Project="$(MSBuildThisFileDirectory)..\Build\Workarounds.props" />
 
-  <Import Condition="'$(_ExtrasHasDesktopSdk)' == 'true' " Project="$(MicrosoftWindowsDesktopSdkPath)\Microsoft.NET.Sdk.WindowsDesktop.props "/>
+  <Import Condition="'$(_ExtrasHasDesktopSdk)' == 'true' and !$(NETCoreSdkVersion.StartsWith('5'))" Project="$(MicrosoftWindowsDesktopSdkPath)\Microsoft.NET.Sdk.WindowsDesktop.props "/>
```

Comment:
This needs to be a version comparison or it won't handle .NET 6, something similar to this:

https://github.com/novotnyllc/MSBuildSdkExtras/blob/main/Source/MSBuild.Sdk.Extras/Build/RIDs.targets#L151

The condition should set the property `ExtrasUseWindowsDesktopApp` to false and the import conditioned on that, as that'll prevent the other items from being added/imported that are related.
Also, th

Reply:
Thank you for the input, I tried to make the suggested changes but I could not get the version parsing to work so I just check the first digit which should be fine for a couple of releases at least.

I am a bit unsure about what to do with `<FrameworkReference Remove="Microsoft.WindowsDesktop.App">` so I removed that when running under 5.0 SDK